accepted/2025/dotnet-pack-nuspec.md

```diff
@@ -0,0 +1,75 @@
+
+#### Example Comparison
+
+|Scenario|Option 1: .nuspec as Argument to `dotnet pack`|Option 2: New Subcommand `dotnet pack nuspec`|
+|--------|----------------------------------------------|---------------------------------------------|
+| Basic Usage | `dotnet pack MyPackage.nuspec --output ./artifacts` | `dotnet pack nuspec MyPackage.nuspec --output ./artifacts` |
+| Example with Property Replacement | `dotnet pack MyPackage.nuspec --property Version=1.2.3 --property Configuration=Release --output ./artifacts` | `dotnet pack nuspec MyPackage.nuspec --property Version=1.2.3 --property Configuration=Release --output ./artifacts` |
+| Relative Path| `dotnet pack ../MyPackage.nuspec --output ./artifacts` | `dotnet pack nuspec ../MyPackage.nuspec --output ./artifacts` |
+
+### Options Relevant to `.nuspec`-only Support in `dotnet pack`
+
+| Option / Flag                  | Description                                                      | Notes / Status for .nuspec-only                                 | Comparison (`nuget pack -h` / `dotnet pack -h`)                |
+|------------------------------- |------------------------------------------------------------------|-----------------------------------------------------------------|---------------------------------------------------------------|
+| `--property` / `-p:`           | Set or override a property for token replacement in `.nuspec`.   | Reused; interpret for token replacement, not just MSBuild.       | `-Properties` in `nuget pack`, `--property` in `dotnet pack`   |
+| `--output`                     | Specifies the output directory for the generated package.         | Reused; already available.                                       | `-OutputDirectory` in `nuget pack`, `--output` in `dotnet pack`|
+| `--basepath`                   | Specifies the base path for resolving relative file paths.        | New; needed for file path resolution in `.nuspec`.               | `-BasePath` in `nuget pack`, not present in `dotnet pack`      |
+| `--version-suffix` / `--version` | Sets the version or version suffix for the package.             | Reused/Clarify; clarify how it applies to `.nuspec` versioning.  | `-Version` in `nuget pack`, `--version-suffix` in `dotnet pack`|
+| `--exclude`                    | Exclude specific files or patterns from the package.              | New; needed for file exclusion support.                          | `-Exclude` in `nuget pack`, not present in `dotnet pack`       |
+| `--nologo`                     | Suppresses display of the startup banner and copyright.           | Reused; already available.                                       | `-NoLogo` in `nuget pack`, `--nologo` in `dotnet pack`         |
+| `--min-client-version`         | Sets the minimum NuGet client version required to install the package. | Consider; may need to add.                                 | `-MinClientVersion` in `nuget pack`, not in `dotnet pack`      |
```

Comment:
can't min client version be set in the nuspec itself?  I don't understand why this would need to be a CLI option